cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -32,7 +32,8 @@ spec:
             - --sink=gcl
             - --sink=gcmautoscaling
```

Comment:
IIUC quota limitations are not about the frequency of writes but rather amount of data in terms of number of datapoints (https://github.com/kubernetes/heapster/blob/master/sinks/gcm/core.go#L223).